src/components/booking/booking-card/BookingCard.vue

```diff
@@ -0,0 +1,333 @@
+<template>
+  <div class="booking-card">
+    <div :class="`wrapper ${isSmallOrDown ? 'p-5' : 'py-6 px-8'}`" @click="mobileEmit">
+
+      <div class="main-info">
+
+        <div class="head">
+          <div class="primary-info">
+            <p class="date txt-1 bold">{{ bookingDate }}</p>
```

Comment:
I use `bold` here because it's the naming convention use by most of the design system provider like [Bootstrap](https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics)
Also, you can have a look [here](https://github.com/fishtripr-com/ui/blob/development/src/assets/style/utils/_variables.scss#L62) to understand how I build it (it's not optimal yet)

Reply:
Also because we have `semibold` to (see [here](https://github.com/fishtripr-com/ui/blob/feature/booking-card/src/assets/style/utils/_classes.scss#L80))

Reply:
I still think we should in that case probably do something like `<strong class="semibold">`

making text bold makes it more important which also means that it should be more important in the semantics of your HTML page.

Reply:
Indeed I misunderstood your remark